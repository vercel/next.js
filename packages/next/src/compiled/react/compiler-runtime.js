/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react-compiler-runtime.production.js');
} else {
  module.exports = require('./cjs/react-compiler-runtime.development.js');
}
