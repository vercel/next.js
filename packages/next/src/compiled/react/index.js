'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react.production.js');
} else {
  module.exports = require('./cjs/react.development.js');
}
