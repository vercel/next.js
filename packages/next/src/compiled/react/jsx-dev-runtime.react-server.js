'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react-jsx-dev-runtime.react-server.production.js');
} else {
  module.exports = require('./cjs/react-jsx-dev-runtime.react-server.development.js');
}
