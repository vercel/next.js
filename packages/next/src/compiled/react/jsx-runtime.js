'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react-jsx-runtime.production.js');
} else {
  module.exports = require('./cjs/react-jsx-runtime.development.js');
}
