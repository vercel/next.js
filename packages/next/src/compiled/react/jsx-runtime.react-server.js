'use strict';

if (process.env.NODE_ENV === 'production') {
  module.exports = require('./cjs/react-jsx-runtime.react-server.production.js');
} else {
  module.exports = require('./cjs/react-jsx-runtime.react-server.development.js');
}
