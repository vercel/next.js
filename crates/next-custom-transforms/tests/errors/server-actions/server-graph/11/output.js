import 'react';
'use sevrer' // ^Tools like Prettier sometimes do this.
;
