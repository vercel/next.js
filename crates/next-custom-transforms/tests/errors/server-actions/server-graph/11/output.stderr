  x The "use server" directive must be at the top of the file, and cannot be wrapped in parentheses.
  | 
   ,-[input.js:2:1]
 1 | import 'react'
 2 | ;('use sevrer')
   :  ^^^^^^^^^^^^^^
   `----
